src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBrokerConstants.cs

```diff
@@ -36,7 +36,7 @@ internal class iOSBrokerConstants
         public const string BrokerKeyLabel = "Broker Key Label";
         public const string BrokerKeyComment = "Broker Key Comment";
         public const string BrokerKeyStorageDescription = "Storage for broker key";
-        public const string LocalSettingsContainerName = "MicrosoftIdentityClient";
+        public const string LocalSettingsContainerName = "ActiveDirectoryAuthenticationLibrary";
```

Comment:
Same questions as @jmprieur asked:

- what is the impact of this change to existing apps? is it a breaking change? will it result in users having to login again?

Reply:
i sent you an email with the details.